<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodingConverterClass</class>
 <widget class="QMainWindow" name="EncodingConverterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncodingConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnSrcDir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>选择源文件目录</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="srcDirText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择源文件编码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cboSrcEn">
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8 with BOM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8 without BOM</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnDstDir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>选择保存目录</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="dstDirText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>选择目标文件编码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="cboDstEn">
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8 with BOM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8 without BOM</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>开始转换</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>内容预览</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actOpenDir"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpenDir">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="encodingconverter.qrc"/>
 </resources>
 <connections/>
</ui>
